driver/driver.go

```diff
@@ -23,31 +28,74 @@ func (d driver) Open(dsn string) (sqldriver.Conn, error) {
 	lastIndex := strings.LastIndex(dsn, "@")
 	seps := []string{dsn[:lastIndex], dsn[lastIndex+1:]}
 	if len(seps) != 2 {
-		return nil, errors.Errorf("invalid dsn, must user:password@addr[?db]")
+		return nil, errors.Errorf("invalid dsn, must user:password@addr[[?db[&param=X]]")
```

Comment:
```suggestion
		return nil, errors.Errorf("invalid dsn, must user:password@addr[?db[&param=X]]")
```

And the comment above the function can be updated